step4/src/service.js

```diff
@@ -0,0 +1,123 @@
+import { template } from './template.js';
+
+export const state = {
+  todoItems: [],
+  selectedItem: -1
+};
+
+const BASE_URL = 'http://localhost:3000';
```

Comment:
```suggestion
const BASE_URL = '/api';
```
api server에 html이 붙어있다고 가정한다면, 굳이 localhost:3000으로 호출하지 않아도 된답니다!